package.json

```diff
@@ -3,6 +3,11 @@
   "version": "1.0.0",
   "private": true,
   "description": "Next-generation OpenFN tooling",
+  "type": "module",
```

Comment:
This repo must not be `type: module`

Reply:
I have already removed the ava.config.mjs and made changes of ava.config.js

Reply:
You have replied to the wrong message.

Please take more time and care over your communications. Everything gets harder when your comments are inaccurate.